### PR TITLE
DDT-1107: TimestampInfo integration tests

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: ui-support-service/pom.xml [hunk 1]
```diff
@@ -57,13 +58,13 @@
...
-            <groupId>org.dcsa.jit</groupId>
-            <artifactId>dcsa-jit-service</artifactId>
-            <version>1.1.0-SNAPSHOT</version>
-            <scope>compile</scope>
-        </dependency>
-    </dependencies>
+		<dependency>
+			<groupId>org.dcsa.jit</groupId>
+			<artifactId>dcsa-jit-service</artifactId>
+			<version>1.1.0-SNAPSHOT</version>
```
Comment (FinnAtAsseco) [inline ui-support-service/pom.xml:64]: version should be in dependency management in parent, not here

**Review by FinnAtAsseco — CHANGES_REQUESTED** [review]

File: ui-support-service/pom.xml [hunk 2]
```diff
@@ -57,13 +58,13 @@
...
+			<scope>compile</scope>
```
Comment (FinnAtAsseco) [inline ui-support-service/pom.xml:65]: scope compile is redundant

Comment (mp-assco) [reply ui-support-service/pom.xml]: removed

File: ui-support-service/pom.xml [hunk 3]
```diff
@@ -57,13 +58,12 @@
...
-            <groupId>org.dcsa.jit</groupId>
-            <artifactId>dcsa-jit-service</artifactId>
-            <version>1.1.0-SNAPSHOT</version>
-            <scope>compile</scope>
-        </dependency>
-    </dependencies>
+		<dependency>
+			<groupId>org.dcsa.jit</groupId>
+			<artifactId>dcsa-jit-service</artifactId>
+			<version>${jit.version}</version>
```
Comment (FinnAtAsseco) [inline ui-support-service/pom.xml:64]: remove version. use the dependencyManagement section of the parent.

**Review by FinnAtAsseco — APPROVED** [review]

**Review by FinnAtAsseco — APPROVED** [review]

**Pull request merged**